app/wsgi.py

```diff
@@ -0,0 +1,13 @@
+"""Entrypoint to the Django WSGI app."""
+
+import os
+import sys
+
+from django.core import wsgi
+
+sys.path.append(os.path.join(os.path.dirname(__file__), 'vendors'))
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'settings')
+
+# pylint: disable=invalid-name
```

Comment:
Just curious: Why is this needed?

Reply:
The naming sounds good, but can you comment the reason of this lint exemption if you stick with it?

Reply:
Done.